### ISSUE TITLE
Clear button in Interface no longer clears Image input (new in release 3.34.0)

### ISSUE THREAD

**Issue by shrdluk** [gradio-app/gradio#4473]
### Describe the bug

Pressing the Clear button in a Gradio app created with Interface doesn't clear the Image to allow a new one to be uploaded. 

After pressing Clear, the app appears to lock up, and the X button in the top right of the image window doesn't work either.

This bug has appeared in release 3.34.0. 

Release 3.33.1 works as expected

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

With the latest version of Gradio installed, run:
```
import gradio as gr

def process_image(img):
    return "Done. Now press Clear and upload another image"

demo = gr.Interface(process_image, gr.Image(), gr.Label()).launch()
```

Upload an image to the Image widget. Then press Clear. Note that the image remains and the X button in the top right of the image window won't clear it either. (The X does work if you press it before Clear).

Now downgrade to an earlier version, eg:
`$ pip install 'gradio==3.33.1' --force-reinstall`

and try the same again.

### Logs

```shell
No errors.
```

### System Info

```shell
Gradio release 3.34.0 (installed via pip)
Mageia 8
Firefox 102.11.0esr
```

### Severity

blocking upgrade to latest gradio version

**Comment by freddyaboulton** [comment]
This is because of the recent changes to Label. If you use Textbox as output, clear button will work. Will fix. Thanks @shrdluk !

**Comment by abidlabs** [comment]
And the fix for Label is coming soon, as part of #4456
